jnosql-elasticsearch-driver/pom.xml

```diff
@@ -42,14 +51,48 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.elasticsearch.client</groupId>
-            <artifactId>elasticsearch-rest-high-level-client</artifactId>
+            <groupId>co.elastic.clients</groupId>
+            <artifactId>elasticsearch-java</artifactId>
             <version>${es.version}</version>
+            <exclusions>
+                <exclusion>
```

Comment:
@dearrudam, why did you remove these dependencies?

If it is because of the Jakarta Big bang, please, remove the TCK test that we'll update next year.

We should not change the driver dependencies.

In general, it is an adapter layer.

Reply:
Fixed